README.md

```diff
@@ -1,21 +1,63 @@
 # logilica-weekly-report
 
-This project exports Logilica data to add it to a weekly report. 
-
-The list of teams is read from the local `weekly_report.yaml` file.
-
-When a new team is onboarded, a new dashboard should be created for it, and the 
-`weekly_report.yaml` file is updated accordingly.
-
-Environment variables must be set for `LOGILICA_DOMAIN`, `LOGILICA_EMAIL`, and
-`LOGILICA_PASSWORD`, the corresponding values can be obtained from Bitwarden.
-
-To run the script checkout the git repo and to setup your environment run 
-`pip install -r src/requirements.txt` 
-
-To run in debug mode:
-`PWDEBUG=1 python3 download_pdfs.py`. For mode details on running and debugging 
-see https://playwright.dev/python/docs/running-tests.
-
-The downloaded PDF files will be stored in the location defined by `Download_path:` in the `weekly_report.yaml` file.
-
+This project exports charts and text from Logilica reports and (will eventually)
+add the content to a Google Doc such as a weekly report.
+
+The tool is configured via a YAML file which contains configuration options as
+well as a list of teams.  For each team, it specifies a list of Logilica
+dashboards and the team's Jira project.  For each dashboard, it specifies the
+name of an output file (which must not conflict with any other team's dashboards'
+output files) and possibly other attributes.  By default, the file
+`weekly_report.yaml` in the current directory is used, but an alternate path can
+be specified on the command line.
+
+To onboard a new team, create a new Logilica dashboard for it and add the team
+and the dashboard information to the configuration file.
+
+Credentials for accessing Logilica are provided by the environment variables,
+`LOGILICA_DOMAIN`, `LOGILICA_EMAIL`, and `LOGILICA_PASSWORD`.  (Note to the
+Developer Practices Team:  the appropriate values for the bot account can be
+obtained from the Bitwarden vault.)
+
+To run the script, check out the Git repo and run:  ```pip install .```  We
+recommend doing this inside a [virtual environment](https://docs.python.org/3/library/venv.html).
+(If you are doing development on the tool, you may want to specify `-e` in the
+`pip install` command.)  After installing the tool, you also need to install
```

Comment:
This installation steps doesn't create the `logilica-weekly-report` script, right? Can we add that instruction also here?

Reply:
Looks like `pip install .` does create the executable script. But for some reason the dependencies are not installed.
I had to run `pip install -r requirements.txt` to install deps.

Reply:
Thanks for checking this, @baijum!  I'm surprised (and disappointed...) that the `pip install .` doesn't install the requirements; I presume that that is due to the fact that we specify them in `requirements.txt` instead of in `pyproject.toml`.

However, I'm not ready to make that change now, so I've updated the `README.md` to add `-r requirements.txt` to the `install` command line.